src/Files.App/Helpers/NavigationHelpers.cs

```diff
@@ -135,38 +132,31 @@ public static async Task<bool> OpenPath(string path, IShellPage associatedInstan
 			bool isHiddenItem = NativeFileOperationsHelper.HasFileAttribute(path, System.IO.FileAttributes.Hidden);
 			bool isDirectory = NativeFileOperationsHelper.HasFileAttribute(path, System.IO.FileAttributes.Directory);
 			bool isReparsePoint = NativeFileOperationsHelper.HasFileAttribute(path, System.IO.FileAttributes.ReparsePoint);
-			bool IsShortcut = path.EndsWith(".lnk", StringComparison.Ordinal) || path.EndsWith(".url", StringComparison.Ordinal);
+			bool isShortcut = associatedInstance.SlimContentPage.SelectedItem is { IsShortcut: true };
```

Comment:
Whoops, instead of referencing `associatedInstance` directly, we should pass it in as a param.

Reply:
There's a few calls to the OpenPath method throughout the application. Is it better to leave as it was, and let a future large refactor take care of it? The `associatedInstance` is needed for the Filesystemhelpers also (which is going to be a service in a future PR?).

Reply:
Seems reasonable to leave as is.